editor/gui/editor_scene_tabs.cpp

```diff
@@ -135,6 +135,15 @@ void EditorSceneTabs::_scene_tab_input(const Ref<InputEvent> &p_input) {
 	}
 }
 
+void EditorSceneTabs::unhandled_input(const Ref<InputEvent> &p_event) {
```

Comment:
```suggestion
void EditorSceneTabs::unhandled_key_input(const Ref<InputEvent> &p_event) {
```
If you are handling only key events, it's better to use `unhandled_key_input()`. Also you should be checking whether `tab_preview_panel` is visible before checking the event, as it's cheaper.

Reply:
Fixed